core/src/main/scala/org/apache/spark/deploy/history/EventLogFileReaders.scala

```diff
@@ -218,12 +219,24 @@ class RollingEventLogFilesFileReader(
 
   private lazy val eventLogFiles: Seq[FileStatus] = {
     val eventLogFiles = files.filter(isEventLogFile).sortBy { status =>
-      getIndex(status.getPath.getName)
```

Comment:
`RollingEventLogFilesWriter` contains now functions for `AppStatusFile`, `EventLogDir`, `EventLogFile`, `AppStatusFile` and at the end `getIndex` which only works on event files. I'm not insisting the suggested approach but somehow it would be good to make order here.

Reply:
No strong opinion. I'll change it.